src/chrome/content/rules/Washington_Times.xml

```diff
@@ -49,13 +49,15 @@
 
 		Self-signed certificate chain error:
 		- paywaytest.washingtontimes.com
+
+		Non-2xx status code:
+		- contribution.washingtontimes.com
```

Comment:
You can try with `http://contribution.washingtontimes.com/sites/all/themes/nde/logo.png`